debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresSchemaIT.java

```diff
@@ -88,6 +90,16 @@ public void shouldLoadSchemaForBuiltinPostgresTypes() throws Exception {
                               Schema.OPTIONAL_STRING_SCHEMA, Schema.OPTIONAL_STRING_SCHEMA);
             assertTableSchema("public.Quoted_Table", "Quoted_Text_Column",
                               Schema.OPTIONAL_STRING_SCHEMA);
+            assertTableSchema("public.array_table", "int_array, bigint_array, text_array",
+                              SchemaBuilder.array(Schema.OPTIONAL_INT32_SCHEMA).optional().build(), SchemaBuilder.array(Schema.OPTIONAL_INT64_SCHEMA).optional().build(),
+                              SchemaBuilder.array(Schema.OPTIONAL_STRING_SCHEMA).optional().build());
+            /*
+            assertTableSchema("public.composite_table", "c_type",
```

Comment:
This comment should be removed now that complex types are left out of the picture.